tests/integration/standard/test_connection.py

```diff
@@ -172,8 +172,8 @@ def fetch_connections(self, host, cluster):
                     if conn._connections is not None and len(conn._connections) > 0:
                         connections.append(conn._connections)
                 else:
-                    if conn._connection is not None:
-                        connections.append(conn._connection)
+                    if conn._connections and len(conn._connections.values()) > 0:
```

Comment:
1. you can do `len` directly on a `dict`.
2. Pythonic would be `if self._connections:` which tests for both `is not None` and `dict has something in it`.

Reply:
Yes you are right,

I think I wasn't fully awake when fixing this test ;)